apps/studio/src/lib/db/clients/oracle.ts

```diff
@@ -59,9 +59,7 @@ export class OracleClient extends BasicDatabaseClient<DriverResult> {
   readOnlyMode: boolean
 
   constructor(server: IDbConnectionServer, database: IDbConnectionDatabase) {
-    super(knexLib({ client: 'oracledb'}))
-    this.server = server
-    this.database = database
+    super(knexLib({ client: 'oracledb'}), NoOpContextProvider, server, database);
```

Comment:
NIT: You should put `ContextProvider` at the end of the constructor arguments, then it can just have a default of `NoOpContextProvider` and we don't need to explicitly provide it in all the drivers.